usr.bin/ipcs/ipc.c

```diff
@@ -191,6 +200,19 @@ kget(int idx, void *addr, size_t size)
 			errx(1, "%s: %s", symn, kvm_geterr(kd));
 		if ((unsigned)kvm_read(kd, kaddr, addr, size) != size)
 			errx(1, "%s: %s", symn, kvm_geterr(kd));
+
+		if (idx == X_MSQIDS) {
```

Comment:
Maybe go ahead and make this a `switch (idx)` similar to the above so it's simpler to extend in the future if needed?